flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/services/BigQueryServices.java

```diff
@@ -53,7 +55,18 @@ public interface BigQueryServices extends Serializable {
      * @return a storage read client object.
      * @throws IOException
      */
-    StorageReadClient getStorageClient(CredentialsOptions credentialsOptions) throws IOException;
```

Comment:
this is a bit nit picky but these "get" methods seem to be more like "create" methods. i wouldnt expect for a "get" call to throw an IO exception

Reply:
Changed.

Reply:
thanks! you can resolve this convo